File: business/src/main/java/com/isystk/sample/domain/dto/DomaDtoImpl.java
package com.isystk.sample.domain.dto;

import com.isystk.sample.domain.dao.DefaultEntityListener;
import java.io.Serializable;
import org.seasar.doma.Entity;

@SuppressWarnings("serial")
@Entity(listener = DefaultEntityListener.class) // 自動的にシステム制御項目を更新するためにリスナーを指定する
public abstract class DomaDtoImpl implements DomaDto, Serializable {

}

File: business/src/main/kotlin/com/isystk/sample/domain/dto/DomaDtoImpl.kt
package com.isystk.sample.domain.dto

import com.isystk.sample.domain.dao.DefaultEntityListener
import org.seasar.doma.Entity
import java.io.Serializable

@Entity(listener = DefaultEntityListener::class) // 自動的にシステム制御項目を更新するためにリスナーを指定する
abstract class DomaDtoImpl : DomaDto, Serializable